packages/new-polymath-issuer/src/components/DividendList/Presenter.tsx

```diff
@@ -52,12 +54,17 @@ export const DividendListPresenter = ({
           </sc.NewDividendButton>
         </Fragment>
       ) : (
-        <sc.PlaceholderButton href={newDividendUrl}>
+        <sc.PlaceholderButton
+          RouterLink={Link}
```

Comment:
`RouterLink` is going to default to the right `Link` if left out. (react-router-dom's or redux-little-router's depending on the dApp)

Reply:
(So we should be able to remove these)